Roslyn.sln

```diff
@@ -373,6 +373,8 @@ Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Perf.Utilities", "src\Test\
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Perf.Runner", "src\Test\Perf\Runner\Perf.Runner.csproj", "{1AA6D2F0-2C40-4AF6-BB79-50AFDCC62720}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "CompilersPackage", "src\Compilers\CompilersPackage\CompilersPackage.csproj", "{DF4F4B6B-7408-4A2B-B82E-60C819D83271}"
```

Comment:
It's for installing the compiler in "I want to F5 the changes I just made" scenarios.